sys/net/link_layer/ieee802154/Kconfig

```diff
@@ -109,8 +109,68 @@ if KCONFIG_USEMODULE_IEEE802154
         default 4
 
     config IEEE802154_AUTO_ACK_DISABLE
-        bool "Disable Auto ACK support" if !USEPKG_OPENWSN
-        default y if USEPKG_OPENWSN
+        bool "Disable Auto ACK support" if (!USEPKG_OPENWSN && !USEPKG_OPENDSME)
+        default y if (USEPKG_OPENWSN || USEPKG_OPENDSME)
+
+    config IEEE802154_DSME_CAP_REDUCTION
+        bool "Enable CAP reduction" if USEPKG_OPENDSME
+        default n if USEPKG_OPENDSME
+
+    config IEEE802154_DSME_SUPERFRAME_ORDER
+        int "Set the DSME superframe order" if USEPKG_OPENDSME
+        default 3
+        help
+            The SO sets the slot duration to `60 * symbol_time_us * 2^SO` usecs.
+            E.g for O-QPSK an SO=3 renders a slot duration of `60 * 16 * 8` = 7.6 ms and
+            a superframe duration of 122.88 ms.
+
+            Settings this value to 3 allows to transmit full IEEE 802.15.4 O-QPSK frames
+            (127 bytes).
+
+    config IEEE802154_DSME_MULTISUPERFRAME_ORDER
+        int "Set the DSME multisuperframe order" if USEPKG_OPENDSME
+        default 3
+        help
+            The MO sets the number of superframes per multisuperframe to `2^(MO-SO)`.
+
+    config IEEE802154_DSME_BEACON_ORDER
+        int "Set the DSME beacon order" if USEPKG_OPENDSME
+        default 3
+        help
+            The BO sets the beacon interval to `2^(BO-SO)` superframes.
+
+    config IEEE802154_DSME_STATIC_GTS
+        bool "Use static GTS allocation" if USEPKG_OPENDSME
+        default 0
+        help
+            When set, the MAC implementation will not use scheduling functions.
+            This requires that the upper layer allocates slots manually using NETOPT_GTS_ALLOC.
+
+    config IEEE802154_DSME_GTS_EXPIRATION
```

Comment:
depends on `IEEE802154_DSME_STATIC_GTS`?

Reply:
This expiration time of a Guaranteed Time Slot is independent of the allocation method, but the "STATIC_GTS" configuration renews the expiration timer on every slot (so slots will never expire).
This configuration is indirectly ignored when using static GTS allocation.